src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -129,12 +129,12 @@ private static string GetBenvArgs(Dictionary<string, string> benvArgsMap)
 
                 if (usePlatform)
                 {
-                    var excludedDirs = platform.GetDirectoriesToExcludeFromCopyToIntermediateDir();
+                    var excludedDirs = platform.GetDirectoriesToExcludeFromCopyToIntermediateDir(context);
                     if (excludedDirs.Any())
                     {
                         directoriesToExcludeFromCopyToIntermediateDir.AddRange(excludedDirs);
                     }
```

Comment:
Could you fix the styling warnings in this section?

Reply:
Sure 👍